extensions/wikibase/module/styles/schema-alignment.css

```diff
@@ -171,6 +171,23 @@
   opacity: 0.7;
 }
 
+.wbs-refresh-icon {
+  margin-left: 5px;
+  cursor: pointer;
+  opacity: 0.6;
+}
+
+.wbs-refresh-icon:hover {
+  opacity: 1;
+}
+
+.wbs-refresh-icon.wbs-icon {
```

Comment:
I know we have SVGs defined inline elsewhere, but I think it'd be best to stick with the pattern of putting all images under the `images` directory (in this case `extensions/wikibase/module/images`) and referencing that file when needed. You can see a similar pattern here: https://github.com/OpenRefine/OpenRefine/pull/7530. There are also examples of Wikibase-specific images in this edited file.